src/content/docs/style-guide/writing-docs/writer-workflow/tech-writer-workflow.mdx

```diff
@@ -55,72 +55,81 @@ Need to edit a doc? Use this table to decide where to work!
+
+You only need to rebuild the local site when it crashes, not after every save or commit.
 
 ### Prerequisites [#prereq]
 - [Install Node](https://nodejs.org/en/download/)
 - Install Yarn 
   - `npm install -g yarn`
 
+Node and Yarn are tools used to build the site on your local machine.
+
 ### Build the site [#build]
 1. In your terminal, go to your cloned repo, **docs-website**. For example:
     - `cd ~/Documents/github/docs-website`
 2. Run yarn with the following commands: `yarn && yarn start`
 3. The site will take a few minutes to build. Make yourself some tea or coffee. 
-4. Once it's built, you can access your preview site in your browser by navigating to [http://localhost:8000/](http://localhost:8000/)
-
-## 3. Edit a doc [#edit-doc]
-Once your local environment and branch are set up, you're ready to edit a doc. Check out the style guide for writing guidelines.
-
-1. First, ensure your **Current Branch** in GitHub Desktop is set to the correct branch, not **Develop**.
-2. Navigate to the doc you want to edit in Finder. If I wanted to edit a Python agent doc, I would navigate to: `~/Documents/github/docs-website/src/content/docs/agents/python-agent/hosting-services/python-agent-stackato.mdx`
-3. Edit the doc in your text editor of choice. You should write docs in markdown language. Reference the style guide for help with formatting markdown
-4. Save the file with your edits, then follow the same process for any other docs you wish to edit.
-
-## 4. Commit your changes [#commit-changes]
-Once your edits are done, you can commit them. This stages your changes, which you will later push upstream to Github. By pushing your changes, everyone will have access to your branch and commits. 
-
-1. Navigate to GitHub Desktop. The left column should have a record of all the edits you have made to docs. 
-2. In the bottom left corner, name your commit and add a good description of your edits. It should be descriptive enough to ensure that someone can understand all the changes made by simply scanning this description.
-3. Click **Commit to [yourbranchname]**
+4. Once built, preview the site in your browser by navigating to [http://localhost:8000/](http://localhost:8000/)
 
 ## 5. Publish your commits [#publish-commits]
-
-Once you have committed your changes, you're almost ready to open your Pull Request.
-
-First, you need to ensure your branch is pushed upstream.
+Once you're satisfied with doc, you'll want to push all of your commits upstream. This makes them available to everyone else in the GitHub repository. 
 
 1. On GitHub desktop, click the blue **Publish Branch** button if available.
 2. If you don't see the Publish Branch, click the blue **Push Origin** button. 
 
-This will push all your commits upstream and make them available to everyone else through the GitHub repository.
+Now you're ready for a Pull Request. 
```

Comment:
I'm not quite sure what this sentence is telling me :) In what way am I ready? Emotionally? Practically? :) Am I ready to create a pull request? 

I think you can delete this sentence or clarify it a bit.

Reply:
I wanted something to cushion the two chunks of procedural steps and prep Reader for yet another set of procedures. A sentence that says, "Cool, you did many Things... but you're not done yet."